.github/workflows/main.yml

```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
     branches:
     - '*'
+# Adding the manual launch of a github workflow
+  workflow_dispatch:
```

Comment:
What's the use-case for that?

Reply:
To be able to launch it manually or via scripts eg: https://github.com/bensuperpc/PyGithub-examples/blob/main/run_manual_workflow.py

Reply:
> What's the use-case for that?

To be able to launch it manually or via scripts eg: https://github.com/bensuperpc/PyGithub-examples/blob/main/run_manual_workflow.py

Reply:
But why would we do that if it's not in a PR and hence includes no changes at all? :thinking:

Reply:
> But why would we do that if it's not in a PR and hence includes no changes at all?
I will do a pull request with the scripts, It's mostly I don't know if it's relevant to put the scripts, why not

Reply:
> But why would we do that if it's not in a PR and hence includes no changes at all?

I will do a pull request with the scripts, It's mostly I don't know if it's relevant to put the scripts, why not ^^